examples/hello-flake/flake.nix

```diff
@@ -0,0 +1,27 @@
+{
+  description = "An example devShell with flakes";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    rust-overlay = {
+      url = "github:oxalica/rust-overlay";
```

Comment:
For stable toolchains, you can do
```diff
-    url = "github:oxalica/rust-overlay";
+    url = "github:oxalica/rust-overlay/stable";
```
I believe this reduces download size, but haven't checked lately so can't tell you for sure.

Reply:
True, stable branch would result in a smaller download size as the manifests for any nightly versions released since the latest stable would not be included. However, the download size reduction is insignificant enough that it wouldn't be worth increasing the complexity of reading the template for new users.

Reply:
> For stable toolchains, you can do
> 
> ```diff
> -      url = "github:oxalica/rust-overlay";
> +      url = "github:oxalica/rust-overlay/stable";
> ```
> 
> I believe this reduces download size, but haven't checked lately so can't tell you for sure.

I didn't explicitly document the `stable` branch because I have not decided what's the goal for it. Please do not use it for production. Currently, It's just updated less frequently but with the same content (so the same size).